centaur/src/main/resources/integrationTestCases/germline/joint-discovery-gatk/joint-discovery-gatk4-part1.wdl

```diff
@@ -0,0 +1,997 @@
+  File axiomPoly_resource_vcf_index
+  File dbsnp_resource_vcf = dbsnp_vcf
+  File dbsnp_resource_vcf_index = dbsnp_vcf_index
+
+  # ExcessHet is a phred-scaled p-value. We want a cutoff of anything more extreme
+  # than a z-score of -4.5 which is a p-value of 3.4e-06, which phred-scaled is 54.69
+  Float excess_het_threshold = 54.69
+  Float snp_filter_level
+  Float indel_filter_level
+  Int SNP_VQSR_downsampleFactor
+
+  Int num_of_original_intervals = length(read_lines(unpadded_intervals_file))
+  Int num_gvcfs = length(read_lines(sample_name_map))
+
+  # Make a 2.5:1 interval number to samples in callset ratio interval list
+  Int possible_merge_count = floor(num_of_original_intervals / num_gvcfs / 2.5)
+  Int merge_count = if possible_merge_count > 1 then possible_merge_count else 1
+
+  call DynamicallyCombineIntervals {
+    input:
+      intervals = unpadded_intervals_file,
+      merge_count = merge_count,
+      docker_image = python_docker
+  }
+
+  Array[String] unpadded_intervals = read_lines(DynamicallyCombineIntervals.output_intervals)
+
+  scatter (idx in range(length(unpadded_intervals))) {
+    # the batch_size value was carefully chosen here as it
+    # is the optimal value for the amount of memory allocated
+    # within the task; please do not change it without consulting
+    # the Hellbender (GATK engine) team!
+    call ImportGVCFs {
+      input:
+        sample_name_map = sample_name_map,
+        interval = unpadded_intervals[idx],
+        workspace_dir_name = "genomicsdb",
+        disk_size = medium_disk,
+        docker_image = gatk_docker,
+        gatk_path = gatk_path,
+        batch_size = 50
+    }
+
+    call GenotypeGVCFs {
+      input:
+        workspace_tar = ImportGVCFs.output_genomicsdb,
+        interval = unpadded_intervals[idx],
+        output_vcf_filename = "output.vcf.gz",
+        ref_fasta = ref_fasta,
+        ref_fasta_index = ref_fasta_index,
+        ref_dict = ref_dict,
+        dbsnp_vcf = dbsnp_vcf,
+        disk_size = medium_disk,
+        docker_image = gatk_docker,
+        gatk_path = gatk_path
+    }
+
+    call HardFilterAndMakeSitesOnlyVcf {
+      input:
+        vcf = GenotypeGVCFs.output_vcf,
+        vcf_index = GenotypeGVCFs.output_vcf_index,
+        excess_het_threshold = excess_het_threshold,
+        variant_filtered_vcf_filename = callset_name + "." + idx + ".variant_filtered.vcf.gz",
+        sites_only_vcf_filename = callset_name + "." + idx + ".sites_only.variant_filtered.vcf.gz",
+        disk_size = medium_disk,
+        docker_image = gatk_docker,
+        gatk_path = gatk_path
+    }
+  }
+
+  call GatherVcfs as SitesOnlyGatherVcf {
+    input:
+      input_vcfs_fofn = write_lines(HardFilterAndMakeSitesOnlyVcf.sites_only_vcf),
+      output_vcf_name = callset_name + ".sites_only.vcf.gz",
+      disk_size = medium_disk,
+      docker_image = gatk_docker,
+      gatk_path = gatk_path
+  }
+
+#  call SNPsVariantRecalibratorCreateModel {
```

Comment:
Do you intend to keep this commented out?  If so I'd recommend deleting it unless it is serving some other kind of instructive purpose

Reply:
Originally Joint discovery was split into two workflows due to the whole workflow exceeding Travis's 2hour limit. Once the adjustment was made to increase travis's execution limit the whole workflow was able to pass. 
I thought part1 and 2 could be left as is and ignored incase the time limit error pops up again, but perhaps they can be remove entirely and have test be dependent on the original workflow.  I'll remove them, unless anyone objects